src/graph/entities/qg_node.c

```diff
@@ -73,6 +74,20 @@ inline uint QGNode_LabelCount
 	return array_len(n->labels);
 }
 
+inline uint QGNode_MandatoryLabelCount
```

Comment:
this function isn't "short" enough and isn't common enough to justify it being inlined.